test/test_utils.py

```diff
@@ -911,35 +897,21 @@ def test_same_with_or_without_preprocessor(estimator, build_dataset):
                                    dataset_formed.data,
                                    random_state=SEED)
 
-  def make_random_state(estimator):
-    rs = {}
-    if estimator.__class__.__name__[-11:] == '_Supervised':
-      rs['random_state'] = check_random_state(SEED)
-    return rs
-
```

Comment:
This was used to set the `random_state` on `fit`, which is now deprecated. So now it is not necessary and actually was a problem since SCML doesn't have a `random_state` on `fit`.

Reply:
Nice catch again

Reply:
But maybe it is good to fix the random state for everyone at init then?

Reply:
I am not sure what you mean, the init of the estimator? The seed is already fixed by `set_random_state`!

Reply:
Indeed, nevermind :-)